src/content/docs/en/guides/images.mdx

```diff
@@ -530,6 +530,12 @@ import rocket from '../assets/rocket.png';
 
 # My MDX Page
 
+// Local image stored in the src/pages/
+![Houston in the wild](houston.png)
+
+// Local image stored in the src/pages/foo/
+![Houston in the dense forests](foo/houston.png)
```

Comment:
```suggestion
// Local image stored in the the same folder
![Houston in the wild](houston.png)
```
I don't think we should show the second example because it's not really common that someone would keep images in `src/pages/otherfolder/` anyway. I think just showing the same folder is fine, since if you were going to keep images in a different folder, you'd probably NOT put that folder in pages. It just looks weird. But at least this way, collocated images are shown.

Reply:
Alright 👍🏻